src/test/unit/algorithms/SpatialPoolerTest.cpp

```diff
@@ -70,6 +70,13 @@ bool check_vector_eq(UInt arr[], vector<UInt> vec) {  //TODO replace with ArrayB
   return true;
 }
 
+bool check_vector_eq(UInt arr[], vector<SynapseIdx> vec) {
```

Comment:
Where is this used?

Reply:
the SP Test uses a number of variations of those. I added a TODO to improve that, but for now I only add another declaration for the version used in these tests.